text/0000-rustdoc-types-maintainers.md

```diff
@@ -0,0 +1,126 @@
+
+The update/publishing process will be moved to T-Rustdoc. In the medium term, I (`@aDotInTheVoid`) will still do it, but
+- In an offical capacity
+- With bus factor for when I stop.
+
+## Actual Mechanics of the move
+
+### Github
+
+Github has a [list of requirements](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) for transfering repositories.
+
+
+- When you transfer a repository that you own to another personal account, the new owner will receive a confirmation email. The confirmation email includes instructions for accepting the transfer. If the new owner doesn't accept the transfer within one day, the invitation will expire.
+   - N/A: Not transfering to 
+- To transfer a repository that you own to an organization, you must have permission to create a repository in the target organization.
+   - I (`@aDotInTheVoid`) don't have create-repo perms in the `rust-lang` org. Therefore I'll add a member of T-Infra as an owner to `aDotInTheVoid/rustdoc-types` (I can't add teams, as it's not in an org). They'll then move it to the repo to the `rust-lang` org. Once moved, T-Infra can become owners.
+- The target account must not have a repository with the same name, or a fork in the same network.
+   - OK.
+- The original owner of the repository is added as a collaborator on the transferred repository. Other collaborators to the transferred repository remain intact.
+   - OK. After the transfer. T-Rustdoc should be added as a colaborator, and I should be removed so that I only have permissions via my membership in T-Rustdoc. 
+- Single repositories forked from a private upstream network cannot be transferred.
+   - OK.
+
+At the end of this we should have a crate in the rust-lang github org with T-Rustdoc as contributors, and T-infra as owners.
+
+### crates.io
+
+crates.io ownership is managed [via the command line](https://doc.rust-lang.org/cargo/reference/publishing.html#cargo-owner).
+
+I will run the following commands to move ownership.
+
+```
+cargo owner --add github:rust-lang:owners
+cargo owner --add rust-lang-owner
+cargo owner --remove aDotInTheVoid
+```
+
+The `rust-lang-owner` is needed because team owners cannot add new owners. 
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- Adds additional maintenance burden to rustdoc team.
+- One-time maintenance burden to infra team to support move.
+
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- We could keep `rustdoc-types` as a personal project. This preserves the status quo (and is what will happen if this RFC (or something similar) isn't addopted). This is undesirable because
```

Comment:
Typo?

```suggestion
- We could keep `rustdoc-types` as a personal project. This preserves the status quo (and is what will happen if this RFC (or something similar) isn't adopted). This is undesirable because
```